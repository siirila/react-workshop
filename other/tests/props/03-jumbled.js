import React from 'react'
import {render} from 'enzyme'
const Example = require(`../../../src/props/${process.env.TEST_DIR ||
  'exercises'}/03-jumbled`).Example

test('Renders "The expected list of X-Men"', () => {
  expect(render(<Example>Passed in a child string</Example>)).toMatchSnapshot()
})
