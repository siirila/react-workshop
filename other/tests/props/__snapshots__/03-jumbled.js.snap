// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders "The expected list of X-Men" 1`] = `
<div>
  <p>
    The members of the X-Men Gold team are:
  </p>
  <ul>
    <li>
      <a
        href="http://x-men.wikia.com/wiki/Wolverine"
      >
        Wolverine
      </a>
       aka Logan
    </li>
    <li>
      <a
        href="http://x-men.wikia.com/wiki/Cyclops"
      >
        Cyclops
      </a>
       aka Scott Summers
    </li>
    <li>
      <a
        href="http://x-men.wikia.com/wiki/Jean_Grey"
      >
        Jean Grey
      </a>
       aka Jean Summers
    </li>
    <li>
      <a
        href="http://x-men.wikia.com/wiki/Beast"
      >
        Beast
      </a>
       aka Hank McCoy
    </li>
    <li>
      <a
        href="http://x-men.wikia.com/wiki/Storm"
      >
        Storm
      </a>
       aka Ororo Munroe
    </li>
  </ul>
</div>
`;
